experimental/CliffordAlgOrd/src/IO.jl

```diff
@@ -0,0 +1,73 @@
+##################################################
+#
+#  Clifford algebra and elements
+#
+##################################################
+ 
+##### Algebra #####
+function Base.show(io::IO, ::MIME"text/plain", C::CliffordAlgebra)
+  io = pretty(io)
+  print(io, "Clifford algebra of quadratic space with Gram matrix\n")
+  print(io, Indent())
+  show(io, "text/plain", gram_matrix(C))
+  print(io, Dedent(), "\ndefined over $(base_ring(C))")
+end
+ 
+function Base.show(io::IO, C::CliffordAlgebra)
+  if is_terse(io)
+    print(io,"Clifford algebra")
```

Comment:
```suggestion
    print(io, LowercaseOff(), "Clifford algebra")
```
maybe needs `io = pretty(io)` before the `if`